pages/docs/about.mdx

```diff
@@ -1,207 +1,21 @@
 ---
 layout: 'docs'
-title: 'Was ist der IoT Hub?'
+title: 'Über dieses Projekt'
 metaTitle: 'About'
-metaDescription: 'Der Iot Hub ist cool'
+metaDescription: 'Mehr Hitnergrundinformation zum IoT hub'
```

Comment:
Small typo, should be: "Hintergrundinformationen". Also, new name should be used here.